src/js/events/change-state-event.ts

```diff
@@ -1,22 +1,23 @@
 import { STATE_IDLE, STATE_LOADING, STATE_STALLED, STATE_BUFFERING, STATE_COMPLETE, STATE_ERROR } from 'events/events';
+import { CoreModel } from 'types/generic.type';
```

Comment:
this stops us from having to `ts` the model by placing the type def in a different place but i'm not in love with it

Reply:
Maybe we should create a ticket for these or push back typing of some files until others are complete first.

Reply:
Added section for tech debt to our standards tracking as a way to avoid tickets until we understand the scope of the tech debt 👍